ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```diff
@@ -354,11 +369,12 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
       OCP_ARCH: arm64
+      TEST_ADDITIONAL: Network_Observability
     test:
-    - chain: openshift-e2e-test-qe
+    - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-efs
-- as: aws-ipi-ovn-efs-amd-f28-destructive-ui
-  cron: 5 23 13 * *
+- as: aws-ipi-ovn-efs-f7-ui
```

Comment:
Shall we name it `aws-ipi-ovn-efs-arm-f7-ui` ?

Reply:
ah, good point.

Reply:
Done.